core/src/main/resources/org/elasticsearch/bootstrap/security.policy

```diff
@@ -131,4 +131,7 @@ grant {
   permission java.io.FilePermission "/sys/fs/cgroup/cpuacct/-", "read";
   permission java.io.FilePermission "/sys/fs/cgroup/memory", "read";
   permission java.io.FilePermission "/sys/fs/cgroup/memory/-", "read";
+
+  // nocommit: needed by plugin service for extensible plugins, move to a separate jar
```

Comment:
do we want to add a elastic-commons jar to do that?

Reply:
After discussing with @jasontedor this is my plan (although a jar on its own with the extension class loader, not in a general commons jar, since I want to reduce what code is allowed the extra permission to the bare minimum).